opm/simulators/wells/WellContainer.hpp

```diff
@@ -45,15 +45,27 @@ namespace Opm {
 template <class T>
 class WellContainer {
 public:
-
     WellContainer() = default;
 
+    WellContainer(std::size_t size)
+        : m_data(size)
+    {}
```

Comment:
Would you mind marking this as `explicit` please?

Reply:
i'll do even better. i forgot to update my git on the laptop when i did the gpmaint_state fix and thus pushed an old version of this branch. this constructor is in fact not needed at all, and the groupstate one can be defaulted.